### PR TITLE
Add info about customizing entity_picture for a person

### PR DESCRIPTION
**Pull request by JeffLIrion**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10120"></a>

### CODE REVIEW

File: source/_components/person.markdown [hunk 1]
```diff
@@ -86,3 +86,15 @@ person:
...
       - device_tracker.beacon
 ```
+
+### Customizing the picture for a person
+
+You can customize the picture used for a person entity in the `customize:` section of your configuration. For example:
+
+```yaml
+customize:
+  person.ada6789
```
Comment (frenck) [inline source/_components/person.markdown]: invalid yaml

File: source/_components/person.markdown [hunk 2]
```diff
@@ -86,3 +86,15 @@ person:
...
+    entity_picture: "/local/ada.JPG"
```
Comment (frenck) [inline source/_components/person.markdown]: `.JPG` is a weird file extension, please use lowercase.

**Review by frenck — CHANGES_REQUESTED** [review]
I find this a bit weird to add here.
Every entity can be customized this way. Why add it here? And not on every single page we have?

Furthermore, it is documented here: https://www.home-assistant.io/docs/configuration/customizing-devices/#entity_picture

File: source/_components/person.markdown (hunk 1 above)
Comment (JeffLIrion) [reply source/_components/person.markdown]: Fixed

File: source/_components/person.markdown (hunk 2 above)
Comment (JeffLIrion) [reply source/_components/person.markdown]: Fixed

**Comment by JeffLIrion** [comment]
> I find this a bit weird to add here.
> Every entity can be customized this way. Why add it here? And not on every single page we have?
> 
> Furthermore, it is documented here: https://www.home-assistant.io/docs/configuration/customizing-devices/#entity_picture

My reason for adding it here is because I would guess that this is the most common case where users want to customize the picture. I think `picture` should be a configuration parameter for the person component, but until then users will need to customize the entity in this way. 

Would you rather add the link to the section that you provided instead of adding a section describing how to customize the `entity_picture`? If so, I'll make that change.

**Comment by frenck** [comment]
@JeffLIrion Agreed, however, I do think we should at least link to the customization documentation.

**Comment by JeffLIrion** [comment]
Link added!

**Review by frenck — APPROVED** [review]
Thanks, @JeffLIrion! 👍

**Pull request merged**